modules/nw-ptp-operator-metrics-reference.adoc

```diff
@@ -23,9 +18,38 @@ Some of the following metrics are applicable for PTP grandmaster clocks (T-GM) o
 |`openshift_ptp_clock_class`
 |Returns the PTP clock class for the interface.
 Possible values for PTP clock class are 6 (`LOCKED`), 7 (`PRC UNLOCKED IN-SPEC`), 52 (`PRC UNLOCKED OUT-OF-SPEC`), 187 (`PRC UNLOCKED OUT-OF-SPEC`), 135 (`T-BC HOLDOVER IN-SPEC`), 165 (`T-BC HOLDOVER OUT-OF-SPEC`), 248 (`DEFAULT`), or 255 (`SLAVE ONLY CLOCK`).
-Applicable to T-GM clocks only.
 |`{node="compute-1.example.com",process="ptp4l"} 6`
 
+|`openshift_ptp_process_restart_count`
+|Returns a count of the number of times the `ptp4l` process was restarted.
```

Comment:
This is just not ptp4l process. It is coudn ot ptp4l, phc2sys , ts2phc etc .
YOu could say number of times the ptp process managed by linuxptp-daemon-conatiner for an example ,ptp4l, ts2phc, phc2sys , gpsd etc